build/Targets/Dependencies.props

```diff
@@ -17,74 +17,75 @@
     <MicrosoftCompositionVersion>1.0.27</MicrosoftCompositionVersion>
     <MicrosoftDiagnosticsRuntimeVersion>0.8.31-beta</MicrosoftDiagnosticsRuntimeVersion>
     <MicrosoftDiagnosticsTracingTraceEventVersion>1.0.35</MicrosoftDiagnosticsTracingTraceEventVersion>
-    <MicrosoftDiaSymReaderVersion>1.1.0-beta1-60625-03</MicrosoftDiaSymReaderVersion>
+    <MicrosoftDiaSymReaderVersion>1.1.0-beta1-60818-02</MicrosoftDiaSymReaderVersion>
     <MicrosoftDiaSymReaderNativeVersion>1.5.0-beta1</MicrosoftDiaSymReaderNativeVersion>
-    <MicrosoftDiaSymReaderPortablePdbVersion>1.2.0-beta1-60831-01</MicrosoftDiaSymReaderPortablePdbVersion>
-    <MicrosoftVisualBasicVersion>10.0.1</MicrosoftVisualBasicVersion>
+    <MicrosoftDiaSymReaderPortablePdbVersion>1.2.0-beta1-61104-02</MicrosoftDiaSymReaderPortablePdbVersion>
+    <MicrosoftNETCoreRuntimeCoreCLRVersion>1.1.0</MicrosoftNETCoreRuntimeCoreCLRVersion>
+    <MicrosoftVisualBasicVersion>10.1.0</MicrosoftVisualBasicVersion>
     <RoslynToolsMicrosoftLocateVSVersion>0.2.0-beta</RoslynToolsMicrosoftLocateVSVersion>
     <RoslynToolsMicrosoftSignToolVersion>0.2.0-beta</RoslynToolsMicrosoftSignToolVersion>
     <RoslynToolsMicrosoftVSIXExpInstallerVersion>0.2.1-beta</RoslynToolsMicrosoftVSIXExpInstallerVersion>
-    <SystemAppContextVersion>4.1.0</SystemAppContextVersion>
-    <SystemCollectionsVersion>4.0.11</SystemCollectionsVersion>
-    <SystemCollectionsConcurrentVersion>4.0.12</SystemCollectionsConcurrentVersion>
-    <SystemCollectionsImmutableVersion>1.2.0</SystemCollectionsImmutableVersion>
-    <SystemConsoleVersion>4.0.0</SystemConsoleVersion>
-    <SystemDiagnosticsContractsVersion>4.0.1</SystemDiagnosticsContractsVersion>
-    <SystemDiagnosticsDebugVersion>4.0.11</SystemDiagnosticsDebugVersion>
-    <SystemDiagnosticsFileVersionInfoVersion>4.0.0</SystemDiagnosticsFileVersionInfoVersion>
-    <SystemDiagnosticsProcessVersion>4.1.0</SystemDiagnosticsProcessVersion>
-    <SystemDiagnosticsStackTraceVersion>4.0.1</SystemDiagnosticsStackTraceVersion>
-    <SystemDiagnosticsToolsVersion>4.0.1</SystemDiagnosticsToolsVersion>
-    <SystemDynamicRuntimeVersion>4.0.11</SystemDynamicRuntimeVersion>
-    <SystemGlobalizationVersion>4.0.11</SystemGlobalizationVersion>
+    <SystemAppContextVersion>4.3.0</SystemAppContextVersion>
+    <SystemCollectionsVersion>4.3.0</SystemCollectionsVersion>
+    <SystemCollectionsConcurrentVersion>4.3.0</SystemCollectionsConcurrentVersion>
+    <SystemCollectionsImmutableVersion>1.3.0</SystemCollectionsImmutableVersion>
```

Comment:
CC @jasonmalinowski and @DustinCampbell specifically for this item because it changes binary versions we are inserting into Visual Studio.